irs-policy-store/src/test/java/org/eclipse/tractusx/irs/policystore/services/PolicyStoreServiceTest.java

```diff
@@ -102,7 +104,7 @@ void setUp() {
         final DefaultAcceptedPoliciesConfig defaultAcceptedPoliciesConfig = new DefaultAcceptedPoliciesConfig();
         final String defaultPoliciesStr = """
                 [{
-                    "policyId": "default-policy",
+                    "policyId": "%s",
                     "createdOn": "2024-07-17T16:15:14.12345678Z",
```

Comment:
Why is there a fixed timestamp this is not correct to set is to any time. Please use time when this when default policy is created.

Reply:
@mkanal: 
This is a test, not production code. In production, the timestamp will be taken from the configuration (see #542 where the configuration was changed to JSON, which contains the timestamps, unlike the previous YAML definition that only defined constraints). The administrator can choose any value they like when they configure the default policy / policies in the values.yaml file.